.github/ISSUE_TEMPLATE/documentation_report.md

```diff
@@ -1,61 +1,34 @@
 ---
 name: 📝 Documentation Report
 about: Ask us about docs
-
 ---
+## Documentation report
+<!--- Verify first that your improvement is not already reported on GitHub -->
+<!--- Also test if the latest release and devel branch are affected too -->
+<!--- This form is processed by Ansibot (machine), complete *all* sections as described -->
 
-<!---
-Verify first that your issue/request is not already reported on GitHub.
-THIS FORM WILL BE READ BY A MACHINE, COMPLETE ALL SECTIONS AS DESCRIBED.
-Also test if the latest release, and devel branch are affected too.
-ALWAYS add information AFTER (OUTSIDE) these html comments.
-Otherwise it may end up being automatically closed by our bot. -->
-
-##### SUMMARY
-<!--- Explain the problem briefly -->
-
-##### ISSUE TYPE
- - Documentation Report
+### Summary
+<!--- Explain the problem briefly below -->
 
-##### COMPONENT NAME
-<!--- Insert, BELOW THIS COMMENT, the name of the module, plugin, task or feature.
-Do not include extra details here, e.g. "vyos_command" not "the network module vyos_command" or the full path-->
+### Component name
+<!--- Write the name of the module, plugin, task or feature below -->
 
-##### ANSIBLE VERSION
-<!--- Paste, BELOW THIS COMMENT, verbatim output from "ansible --version" between quotes below -->
-```
+### Ansible version
+<!--- Paste verbatim output from "ansible --version" between quotes -->
+```paste below
 
 ```
 
-##### CONFIGURATION
-<!--- If using Ansible 2.4 or above, paste, BELOW THIS COMMENT, the results of "ansible-config dump --only-changed"
-Otherwise, mention any settings you have changed/added/removed in ansible.cfg
-(or using the ANSIBLE_* environment variables).-->
-
-##### OS / ENVIRONMENT
-<!--- Mention, BELOW THIS COMMENT, the OS you are running Ansible from, and the OS you are
-managing, or say "N/A" for anything that is not platform-specific.
-Also mention the specific version of what you are trying to control,
-e.g. if this is a network bug the version of firmware on the network device.-->
-
-##### STEPS TO REPRODUCE
-<!--- For bugs, show exactly how to reproduce the problem, using a minimal test-case.
-For new features, show how the feature would be used. -->
-
-<!--- Paste example playbooks or commands between quotes below -->
-```yaml
+### Configuration
+<!--- Paste verbatim output from "ansible-config dump --only-changed" between quotes -->
+```paste below
 
 ```
 
-<!--- You can also paste gist.github.com links for larger files -->
-
-##### EXPECTED RESULTS
-<!--- What did you expect to happen when running the steps above? -->
+### OS / Environment
```

Comment:
For a docs issue, is the user's local OS/environment relevant?

Reply:
:+1:

Reply:
It might, for the same reason Ansible's version is relevant. It helps to understand why a user wants to improve the docs if we understand where he is coming from. But again, in some cases it is irrelevant too.

Reply:
It's better to have it when we don't need it, than to not have it when we need it.
If it is irrelevant most users will state so, or leave it out.